packages/extension-ui/src/Popup/index.tsx

```diff
@@ -22,6 +22,7 @@ import ImportSeed from './ImportSeed';
 import Settings from './Settings';
 import Signing from './Signing';
 import Welcome from './Welcome';
+import ToastProvider from '../components/Toast/ToastProvider';
```

Comment:
Make ToastProvider analogical to other providers (make ToastContext and ToastContext.Provider)